x-pack/plugins/timelines/server/plugin.ts

```diff
@@ -33,6 +33,7 @@ export class TimelinesPlugin
       const TimelineSearchStrategy = timelineSearchStrategyProvider(
         depsStart.data,
         depsStart.alerting,
+        this.logger,
```

Comment:
The same here. Need to be cleaned up.

Reply:
@YulNaumenko I added this for future use. so that search strategy provider can have access to the logger if needed because I have needed it more than a couple of times.. do you still think this needs to be removed?